airbyte-cdk/python/unit_tests/sources/file_based/scenarios/check_scenarios.py

```diff
@@ -174,8 +174,8 @@
             ],
         }
     )
-    .set_validation_policies(FailingSchemaValidationPolicy)
-    .set_expected_check_error(ConfigValidationError, FileBasedSourceError.ERROR_VALIDATING_RECORD)
+    .set_validation_policies({FailingSchemaValidationPolicy.ALWAYS_FAIL:  FailingSchemaValidationPolicy()})
+    .set_expected_check_error(None, FileBasedSourceError.ERROR_VALIDATING_RECORD)
```

Comment:
This was actually a buggy test. Validation policies should be a mapping of name -> policy. But we passed in just the policy so this actually ended up failing the pydantic parse. When we correct the inputs, we actually want to verify that the `check` fails due to the record failing validation. And in this case its not an explicit exception, its an error message in the check response failure